README.rst

```diff
@@ -1,3 +1,17 @@
+.. |.examples.diffusion.mesh1D| replace:: ``examples.diffusion.mesh1D``
+.. _.examples.diffusion.mesh1D: https://github.com/usnistgov/fypy/blob/2730380faa50d1b2f9cc6d5a1b5d9f9f4a41bd1f/examples/diffusion/mesh1D.py
+.. |.faq| replace:: Frequently Asked Questions
+.. _.faq: https://pages.nist.gov/fipy/en/latest/FAQ.html#faq
+.. |.FiPy| replace:: FiPy
+.. _.FiPy: https://pages.nist.gov/fipy/en/latest/glossary.html#term-FiPy
```

Comment:
Links like `.FiPy` generate [spelling errors](https://dev.azure.com/guyer/FiPy/_build/results?buildId=1607&view=logs&j=590c7ebb-8554-5341-5372-44ea3649a043&t=27e67363-6dd9-5c72-e3dc-8f391dff9547&l=65).

They can be added to the dictionary, but maybe there's a sphinx_readme option that avoids the error?

Reply:
I poked at the [sphinx_readme docs](https://sphinx-readme.readthedocs.io/en/latest/configuration/configuring.html) and sources but couldn't find an option that treats `:term:` or glossary items specially. This is the first time I've really used Restructured Text and Sphinx so I'm pretty cold on it.  It looks like there's a bunch of rules in  [.pyspelling.yml](https://github.com/usnistgov/fipy/blob/master/.pyspelling.yml) that already deal with restructured text - maybe adding a rule there to deal with this class of problems is in order instead.

Reply:
> It looks like there's a bunch of rules in [.pyspelling.yml](https://github.com/usnistgov/fipy/blob/master/.pyspelling.yml) that already deal with restructured text - maybe adding a rule there to deal with this class of problems is in order instead of hard-coding more terms in [.en-custom.txt](https://github.com/usnistgov/fipy/blob/master/.en-custom.txt).

`.pyspelling.yml` is definitely the way to go. Let me see if I can get the rule right.

Reply:
I have this working, although pyspelling's filters are weird and completely undocumented